src/test/mod.rs

```diff
@@ -90,9 +90,15 @@ where
 }
 
 fn is_file_skip(path: &Path) -> bool {
-    SKIP_FILE_WHITE_LIST
-        .iter()
-        .any(|file_path| is_subpath(path, file_path))
+    let filter_file = std::env::var("TEST_FILE").ok();
+    let skip_file = filter_file
+        .map(|filter_file| !path.to_str().unwrap().contains(&filter_file))
+        .unwrap_or(false);
+
+    skip_file
+        || SKIP_FILE_WHITE_LIST
+            .iter()
+            .any(|file_path| is_subpath(path, file_path))
```

Comment:
Interesting idea. It feels a little out of scope for the fix to 5260 though, so I'd like to see this change proposed in a follow up PR. There might be some follow up discussion regarding whether or not this is the approach we want to take to run tests on single files.

Reply:
okay I'll move it to a different PR